README.md

```diff
@@ -11,7 +11,9 @@ Create scale-based layout with a bit more ease. Powered by extensions.
 ```yaml
 dependencies:
   flutter_scale_aware: 
-    git: https://github.com/jogboms/flutter_scale_aware.git
+    git: 
```

Comment:
We could let go of this now as its been merged to master.

Reply:
That makes sense!